clients/src/main/java/org/apache/kafka/common/config/SslConfigs.java

```diff
@@ -110,6 +110,10 @@
     public static final String SSL_SECURE_RANDOM_IMPLEMENTATION_CONFIG = "ssl.secure.random.implementation";
     public static final String SSL_SECURE_RANDOM_IMPLEMENTATION_DOC = "The SecureRandom PRNG implementation to use for SSL cryptography operations. ";
 
+    public static final String SSL_ENGINE_FACTORY_CLASS_CONFIG = "ssl.engine.factory.class";
+    public static final String DEFAULT_SSL_ENGINE_FACTORY_CLASS = "org.apache.kafka.common.security.ssl.DefaultSslEngineFactory";
```

Comment:
`org.apache.kafka.common.security.ssl.DefaultSslEngineFactory.class.getName()` instead of hard-coded string.